aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1844,6 +1888,7 @@ impl AptosSimulationVM {
                                         &mut gas_meter,
                                         &storage_gas_params.change_set_configs,
                                         &txn_data,
+                                        false,
```

Comment:
same

Reply:
As you mentioned, I enabled aggregators for simulate_signed_transaction and disabled aggregators for failure_transaction_cleanup.